src/vs/workbench/parts/extensions/node/extensionsWorkbenchService.ts

```diff
@@ -414,6 +415,10 @@ export class ExtensionsWorkbenchService implements IExtensionsWorkbenchService,
 				if (this.isAutoUpdateEnabled()) {
 					this.checkForUpdates();
 				}
+				if (!this.continuousSyncSetup && this.configurationService.getValue(AutoUpdateConfigurationKey) !== 'off') {
+					this.continuousSyncSetup = true;
```

Comment:
Why is this variable needed ? If the value has changed you should either stop or start automatic checking?

Reply:
If the config setting changed to `off`, then the loop for syncing is stopped. I added this to start the loop again. But on second thoughts, I can achieve the same without adding a new variable. I'll do that